frontend/app/src/composables/settings/accounting/index.ts

```diff
@@ -35,7 +43,47 @@ export const useAccountingSettings = () => {
     }
   };
 
+  const getAccountingRulesConflicts = async (
+    payload: MaybeRef<AccountingRuleConflictRequestPayload>
+  ): Promise<Collection<AccountingRuleConflict>> => {
+    try {
+      const response = await fetchAccountingRuleConflicts(get(payload));
+
+      return mapCollectionResponse(response);
+    } catch (e: any) {
+      logger.error(e);
+      const message = e?.message ?? e ?? '';
+
+      notify({
+        title: t(
+          'accounting_settings.rule.conflicts.fetch_error.title'
+        ).toString(),
```

Comment:
We don't really need `.toString()` here anymore this returns a `string` already instead of a `TranslationResult` or whatever it was before